test/types/community/collection/findX.test-d.ts

```diff
@@ -209,7 +209,7 @@ colorCollection.find<{ color: string }>({ color: { $in: colorsFreeze } });
 colorCollection.find<{ color: string }>({ color: { $in: ['regularArray'] } });
 
 // This is a regression test that we don't remove the unused generic in FindOptions
-const findOptions: FindOptions<{ a: number }> = {};
+const findOptions: FindOptions = {};
 expectType<FindOptions>(findOptions);
```

Comment:
I think this can just be deleted - the comment above says the intention is to make sure we don't remove the generic - we are now intentionally removing the generic

Reply:
Removed.